### PR TITLE
Update common bugs for F35 resolv.conf issue

### PR DESCRIPTION
**Pull request by rvykydal**

### CODE REVIEW

File: docs/common-bugs.rst [hunk 1]
```diff
@@ -478,3 +478,16 @@ Enabling root password SSH login via password.
     This does 100% the same as a rootpw command option would, but unlike the option is quite explicit about what it does
     and easy to spot in a kickstart file. Or even better, use the sshkey command to use a key instead of password,
     making a remote guessing attack essentially impossible.
+
+/mnt/sysroot/etc/resolv.conf issue in Fedora 35.
```
Comment (poncovka) [inline docs/common-bugs.rst:482]: Please, remove the dot at the end.

**Review by poncovka — COMMENTED** [review]
Looks good to me, but I don't think it belongs to `Kickstart issues`. What about adding a new section for network issues?

**Review by VladimirSlavik — COMMENTED** [review]
Agreed with Vendy, otherwise fine. Thank you!

File: docs/common-bugs.rst (hunk 1 above)
Comment (rvykydal) [reply docs/common-bugs.rst]: Done.

**Comment by rvykydal** [comment]
> Looks good to me, but I don't think it belongs to `Kickstart issues`. What about adding a new section for network issues?

Definitely, added.

**Review by VladimirSlavik — APPROVED** [review]
Thanks!

**Review by poncovka — APPROVED** [review]

**Pull request merged**